cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -188,6 +188,21 @@ public void testProgramStartStopStatus() throws Exception {
 
     waitState(dummyMR2, STOPPED);
 
+    // start multiple runs of the map-reduce program
+    startProgram(dummyMR2);
+    startProgram(dummyMR2);
+    verifyProgramRuns(dummyMR2, "running", 1);
+    historyRuns = getProgramRuns(dummyMR2, "running");
+    Assert.assertEquals(2, historyRuns.size());
+
+    // stop all runs of the map-reduce program
+    stopProgram(dummyMR2, 200);
+    waitState(dummyMR2, STOPPED);
```

Comment:
so will the state of the program be STOPPED instead of COMPLETED if it is stopped before it completes? the reason why I ask is that the dummyMR2 program might finish before we try to stop it, so wanted to make sure we are really testing the functionality